include/knowhere/config.h

```diff
@@ -222,11 +224,15 @@ class EntryAccess {
         return *this;
     }
 
+    EntryAccess&
+    for_load() {
+        entry->type |= PARAM_TYPE::LOAD;
+        return *this;
+    }
+
     EntryAccess&
     for_all() {
-        entry->type |= PARAM_TYPE::TRAIN;
-        entry->type |= PARAM_TYPE::SEARCH;
-        entry->type |= PARAM_TYPE::RANGE_SEARCH;
+        entry->type |= PARAM_TYPE::ALL;
```

Comment:
1. I know the naming is confusing, but looks like this `for_all` is only for `train`, `search` and `range search`, not for `feder`. So this change will mess it up.
2. Considering the forward compatibility, let's don't add a enum `ALL` here, or it is easy to trap the following modifier. For instance, I am about to adding one enum for deserialization, and without noticing there is an `ALL`, it is easy to make a mistake.